### PR TITLE
Adding license and copyright information.

### PR DESCRIPTION
**Pull request by jeremiah**
This PR brings in some more formal licensing and copyright information into the README.

Signed-off-by: Jeremiah C. Foster <jeremiah@jeremiahfoster.com>

### CODE REVIEW

**Comment by mardy** [comment]
Do we need to add our copyright line to the python files (and consequently into the README) since we have been working on the code since 2014?

**Comment by jeremiah** [comment]
That might be a good idea. But only on the files we've changed of course.

File: README.md [hunk 1]
```diff
@@ -177,3 +177,12 @@ ADD_CUSTOM_COMMAND (OUTPUT ${GENERATED_STUB}
...
 The usage of the `$GENERATED_STUB` files will trigger the execution of the code
 generator.
+
+## License and copyright
+This README file is Copyright &copy; 2018 Luxoft Sweden AB
+SPDX-License-Identifier: CC-BY-SA-4.0
+[!License: CC BY-SA 4.0](https://img.shields.io/badge/LICENSE-CC--BY--SA--4.0-lightgrey.svg)](https://creativecommons.org/licenses/by-nc/4.0/)
+
+
+The python code is Copyright &copy; 2008-2011 Red Hat, Inc.
```
Comment (mardy) [inline README.md]: Please add a line for Luxoft (looking at the commit history, it looks like we started working on it in 2014).

**Comment by jeremiah** [comment]
I added the Luxoft copyright and removed Jontatan from reviewers since he seems to be on vacation. I'm going to add a 'CONTRIBUTING' file when I get a chance, likely tomorrow.

**Comment by mardy** [comment]
Sorry, totally forgot about this, and the notification got lost :-)

**Pull request merged**